kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/VersionInfo.java

```diff
@@ -26,10 +29,13 @@ public class VersionInfo {
 
   public VersionInfo() {}
 
-  public VersionInfo(String version) {
+  // Explicitly specifies JsonProperty to be compatible if disable auto detect feature
+  @JsonCreator
+  public VersionInfo(@JsonProperty("version") String version) {
```

Comment:
This is the only one dto which is used by trino and kyuubi. Since we disable auto detect feature for Trino, Explicitly specifies JsonProperty here to be compatible both of them.